pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "scikit_build_core.build"
 
 [project]
 name = "spglib"
-version = "2.1.0"
+version = "2.2.0"
 requires-python = ">=3.7"
```

Comment:
python 3.7 is EOL. Should we change it to 3.8? For me, either is fine.

Reply:
Agree should drop `3.7`. Let's do it the first thing after the release